internal/commands/app/create.go

```diff
@@ -65,19 +65,18 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 	}
 
 	var groupID = cmd.inputs.Project
-	if appRemote.IsZero() {
-		if groupID == "" {
-			id, err := cli.ResolveGroupID(ui, clients.Atlas)
-			if err != nil {
-				return err
-			}
-			groupID = id
+
+	if groupID == "" {
+		id, err := cli.ResolveGroupID(ui, clients.Atlas, appRemote.GroupID)
+		if err != nil {
+			return err
 		}
-	} else {
-		groupID = appRemote.GroupID
+		groupID = id
 	}
 
-	err = cmd.inputs.resolveName(ui, clients.Realm, appRemote)
+	err = cmd.inputs.resolveName(ui, clients.Realm, realm.AppFilter{
```

Comment:
Changed this because the resolveName->ResolveApp->FindApps methods was looking for **client app id** while we were trying to find a match for an **app id** through appRemote

Reply:
in the `resolveRemoteApp` implementation, would it help us/make sense to pull `app.ClientAppID` instead of `app.ID`?  honestly the `remoteApp` struct could maybe just go away entirely and we deal with the `realm.App` itself (so we have all that data).  what do you think?